README.md

```
@@ -14,6 +14,7 @@ Check out our [Wiki](https://github.com/Learnathon-By-Geeky-Solutions/binary-bra
 </div>
 
 
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)